client/components/SingleProduct.jsx

```diff
@@ -1,14 +1,59 @@
 import React, {Component} from 'react'
 import {connect} from 'react-redux'
 import {withRouter, Link} from 'react-router-dom'
-import {Jumbotron, Button} from 'react-bootstrap'
+import {Jumbotron, FormControl,Button} from 'react-bootstrap'
 import {getAllProducts} from '../store/products'
 import {updateCart} from '../store/cart'
 import ProductSelector from './ProductSelector'
+import axios from 'axios'
 
 class SingleProduct extends Component {
+  constructor(){
+    super();
+    this.state = {
+      sizes: [],
+      inventory: [],
+      selectedSize: 0,
+      inventoryLeft: -1
+    }
+    this.handleChange = this.handleChange.bind(this)
+  }
+
   async componentDidMount() {
     await this.props.loadProducts()
+    const {data} = await axios.get(`/api/products/size/${this.props.match.params.productId}`);
+    const inventory= await axios.get(`/api/products/quantity/${this.props.match.params.productId}`)
```

Comment:
I think we should move all the axios requests into their own thunk creators and call the dispatched action here instead of directly using axios

Reply:
Yeah i was thinking that as well. Do you want to create an inventory and size in the store?

Reply:
Sure we can talk about it in standup

Reply:
It might take a little longer. I was scared of doing a merge conflict with one of you guys.